build/sb-mirror/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM alpine:latest
 EXPOSE 873/tcp
 
-RUN apk add --no-cache sqlite curl jq rsync 
+RUN apk add --no-cache sqlite curl jq rsync flock 
```

Comment:
flock is already included as a busybox built-in, no?

https://wiki.alpinelinux.org/wiki/BusyBox